automate-sketch.sketchplugin/Contents/Sketch/Artboard/Resize_to_Fit_Height.cocoascript

```diff
@@ -51,3 +54,10 @@ function getRectFromLayers(parentGroup) {
     return rect;
 
 }
+
+function findParentArtboard(layer) {
```

Comment:
When select a layer which is not inside an artboard, the funtcion `findParentArtboard(layer)` will throw an error.

I will change to this.

```javascript
var artboard = layer.parentArtboard();

if (artboard) {
    var rectOfChildLayers = getRectFromLayers(artboard);
    var height = Math.ceil(rectOfChildLayers.y() + rectOfChildLayers.height());
    artboard.frame().setHeight(height);
} else {
    doc.showMessage('"' + layer.name() + '" is not an Artboard or not inside an Artboard.');
}
```

Reply:
Great, thanks. Didn't think of this case. 8-)